### DISCUSSION TITLE
Is coding interactions possible?

### DISCUSSION THREAD

**Discussion by Socialwolfontherun** [olivierfriard/BORIS#421] [Q&A]
Is coding interactions possible in BORIS? So far I have not been able to find studies using BORIS for interactions, nor features in BORIS to facilitate this. What I mean is that for esample I have a subject A doing something involving subject B, and I need to code which individual is the doer and which is the receiver, i.e. the directionality of interaction. Is it possible in BORIS?

**Comment by olivierfriard** [comment]
Hi,

you can define a modifier as a set of receivers for each behavior and code
the doer.
For each behavior that has this modifier BORIS will ask you for the
receiver.
See https://boris.readthedocs.io/en/latest/#set-the-modifiers for details
